pyspark-optimization-tutorial/solution-files/README.md

```diff
@@ -0,0 +1,32 @@
+# Solution Code
+
+This folder contains the complete working solution for the ETL tutorial.
+
+## Try It Yourself First!
+
+Building the pipeline yourself is where the learning happens. Only refer to this solution if:
+- You're completely stuck
+- You want to compare your approach
+- You've finished and want to see alternatives
+
+## Running the Solution
+
+From the project root:
+```bash
+# Copy solution files to root
+cp solution/src/etl_pipeline.py src/
+cp solution/main.py .
```

Comment:
The solution folder is named `solution-files` for this case, we should change either README or the folder name 🙂